app/models/member.rb

```diff
@@ -1,4 +1,6 @@
 class Member < ApplicationRecord
+  include SocialSecurityNumber
```

Comment:
@hartsick including this module is what wait causing crazy failures before. The SocialSecurityNumber concern previously had some overrides to make activemodel act more like activerecord, so when we included it in an activerecord class it blew up.

Solution here was to create a `StepSocialSecurityNumber` concern that includes that stuff and then to have some duplicate logic in a regular `SocialSecurityNumber` concern

Reply:
Ahhh interesting!